libs/wingsdk/src/target-tf-aws/schedule.ts

```diff
@@ -0,0 +1,134 @@
+import { join } from "path";
+import { IamRole } from "@cdktf/provider-aws/lib/iam-role";
+import { IamRolePolicy } from "@cdktf/provider-aws/lib/iam-role-policy";
+import { SchedulerSchedule } from "@cdktf/provider-aws/lib/scheduler-schedule";
+import { Construct } from "constructs";
+import { Function } from "./function";
+import * as cloud from "../cloud";
+import { convertBetweenHandlers } from "../convert";
+import * as core from "../core";
+
+/**
+ * AWS implementation of `cloud.Schedule`.
+ *
+ * @inflight `@winglang/sdk.cloud.IScheduleClient`
+ */
+export class Schedule extends cloud.ScheduleBase {
+  private readonly scheduleExpression: string;
+
+  constructor(scope: Construct, id: string, props: cloud.ScheduleProps = {}) {
+    super(scope, id, props);
+
+    const { rate, cron } = props;
+
+    if (rate && cron) {
+      throw new Error("rate and cron cannot be configured simultaneously.");
+    }
+    if (!rate && !cron) {
+      throw new Error("rate or cron need to be filled.");
+    }
+    if (rate && rate.seconds < 60) {
+      throw new Error("rate can not be set to less than 1 minute.");
+    }
```

Comment:
Can we create an issue about supporting more frequent rates? I think this should be possible with some extra machinery -- see for example https://constructs.dev/packages/cdk-lambda-subminute/v/2.0.109?lang=typescript or https://constructs.dev/packages/paws-lambda-time-trigger/v/1.0.2?lang=typescript

Reply:
Done issue #1551